src/man/sssd.conf.5.xml

```diff
@@ -790,6 +790,32 @@
                         </para>
                     </listitem>
                 </varlistentry>
+                <varlistentry>
+                    <term>refresh_max_random_offset (integer)</term>
+                    <listitem>
+                        <para>
+                            When SSSD is in offline mode it keeps probing
+                            backend servers in specified time intervals:
+                        </para>
+                        <para>
+                            delay = delay + (rand() % random_offset)
```

Comment:
I would expect to see this equation cited under `offline_timeout_max` option. Moreover, I think is best to set the real names of the variables. Example:
`delay = offline_timeout_max + (rand() % refresh_max_random_offset)`

Feel free to fix the example as I'm not completely sure if it is correct.

Reply:
I agree that using real configuration variables here would be more straightforward, will change that.
But the true formula will be more something like:
`delay = ((delay * 2) + (rand() % refresh_max_random_offset)) % offline_timeout_max`

Reply:
Correction:
`delay = (delay * 2) % offline_timeout_max + (rand() % refresh_max_random_offset)`

Reply:
I think this would be the correct equation.

Reply:
I just modified the whole section related to `offline_timeout`, `offline_timeout_max` and `refresh_max_random_offset`. Please take a look and share your thoughts.